src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -112,7 +113,7 @@ To create a NRQL alert condition from a chart:
     src={queriesMetricExplorerAnimation}
   />
 
-4. Follow the steps explained in the [Create a NRQL alert condition from a policy](#alert-condition-from-policy) but be aware that step 1 is a bit different because, now, the query is built with the data you've previously configured.
+4. Follow the steps explained in the [Create a NRQL alert condition from a policy](#alert-condition-from-policy) but be aware that step 2 is a bit different because, now, the query is built with the data you've previously configured.
```

Comment:
```suggestion
4. Follow the steps explained in the [Create a NRQL alert condition from a policy](#alert-condition-from-policy) but be aware that [Step 2](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#step2) is a bit different because, now, the query is built with the data you've previously configured.
```

Reply:
Ooo this is a good idea. I like the link to the specific step!